pkgs/development/python-modules/absl-py/default.nix

```diff
@@ -0,0 +1,27 @@
+{ buildPythonPackage
+, lib
+, fetchurl
+, six
+, pytest
+}:
+
+buildPythonPackage rec {
+  version = "0.1.3";
+  name = "absl-py-${version}";
+
+  src = fetchurl {
+    url = "mirror://pypi/a/absl-py/${name}.tar.gz";
```

Comment:
Please use here in and in the other cases `fetchPypi` instead of `fetchurl`.

Reply:
done